src/components/__tests__/CountdownEvent.cy.js

```diff
@@ -73,46 +83,55 @@ describe('Event Countdown', () => {
   it('renders the numbers in correct format', () => {
     cy.dataCy('countdown-days')
       .should('be.visible')
-      .and('have.css', 'font-size', '64px')
-      .and('have.css', 'font-weight', '700');
+      .and('have.css', 'font-size', '48px')
```

Comment:
CSS property `font-size` and `font-weight` value repeat across _renders the numbers in correct format_ test, you should declare const for these properties values inside _renders the numbers in correct format_ test.

Reply:
Fixed with https://github.com/auto-mat/ride-to-work-by-bike-frontend/pull/308/commits/a1ef472f4d85b283c2bd3a85f3fd3d83470c8e3d.